src/SFA.DAS.Tasks.Domain/Repositories/ITaskRepository.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using SFA.DAS.Tasks.Domain.Entities;
+using Task = SFA.DAS.Tasks.Domain.Entities.Task;
+
+namespace SFA.DAS.Tasks.Domain.Repositories
+{
+    public interface ITaskRepository
+    {
+        Task Create(Task task);
+
+        Task<Task> GetById(long id);
+
+        Task<IList<Task>> GetByAssignee(string assignee);
+
+        TaskAlert Create(TaskAlert taskAlert);
+
+        Task<IList<TaskAlert>> GetByUser(string userId);
```

Comment:
Is it worth changing this to GetAllByUser as it returns a list? The same for GetByAssignee.